PythonPonies/bangazon-api
# updated docstrings for payment type model, serializer, etc

# Description
Add description about what this pull request covers.
Added proper documentation for all methods, arguments, and classes associated with the payment type method.
## Number of Fixes
Type of number of fixes included in this PR.
No fixes.
## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
#1 
## Problem to Solve
Describe the problem(s) this pull request solves.
Adheres to company definition of "done"
## Proposed Changes
Describe the proposed changes.
None
## Expected Behavior
Describe the expected behavior.
Should not affect the functionality of API
## Steps to Test Solution
Fetch the branch and check for comments in models.py, serializers.py, and views.py. Navigate to localhost:8000 and make sure that functionality has not been affected by added comments. 
1. 

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ X] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
